Components/Functional/Footer/index.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Container,
+  Disclaimer,
+  Column,
+  Row,
+  Logo,
+  ColumnName,
+  Anchor,
+  List,
+  Input,
+  InputWrapper,
+} from "@/Components/Styled/Footer";
+import Button from "../Button";
```

Comment:
Remember to use `@/Components/...` import format. I'll clean all of these out in a PR.